core/src/main/scala/org/ensime/indexer/stringymap/StringyMap.scala

```diff
@@ -0,0 +1,283 @@
+// Copyright: 2010 - 2017 https://github.com/ensime/ensime-server/graphs
+// License: http://www.gnu.org/licenses/gpl-3.0.en.html
+/**
+ * TypeClass (api/impl/syntax) for marshalling objects into
+ * `java.util.HashMap<String,Object>` (yay, big data!).
+ */
+package org.ensime.indexer.stringymap
+
+import org.ensime.indexer.orientdb.api.OrientProperty
```

Comment:
hmm, I'd not too keen on making this depend on orientdb's API... perhaps we need a separate typeclass for schemas.